frontend/src/views/Comment.svelte

```diff
@@ -20,17 +20,18 @@
   import Hoverable from "../utils/Hoverable.svelte";
   import Icon from "../utils/Icon.svelte";
   export let comment, rootResource, selfId;
-  let text = comment[1];
-  let addresses = text.matchAll("#[a-fA-F0-9]+[@0x[0-9a-fA-F]+]*", text);
+  let addresses = comment.comment_text.matchAll(
+    "#[a-fA-F0-9]+[@0x[0-9a-fA-F]+]*"
```

Comment:
I know you didn't write this regex originally, but I'm almost certain it doesn't do what it's supposed to do. May as well fix here.

`[xyz]` selects one of the characters `x`, `y`, or `z`, whereas `(xyz)` matches the group of characters `xyz`. This regular expression is almost certainly supposed to match the latter.

```suggestion
    "#[a-fA-F0-9]+(@0x[0-9a-fA-F]+)*"
```

I also wonder if that last `*` is actually supposed to be a `?`...

@dannyp303 any comment on this?

Reply:
`#[a-fA-F0-9]+(@0x[0-9a-fA-F]+)?` seems very reasonable

Reply:
> I know you didn't write this regex originally, but I'm almost certain it doesn't do what it's supposed to do. May as well fix here.
> 
> `[xyz]` selects one of the characters `x`, `y`, or `z`, whereas `(xyz)` matches the group of characters `xyz`. This regular expression is almost certainly supposed to match the latter.
> 
> I also wonder if that last `*` is actually supposed to be a `?`...
> 
> @dannyp303 any comment on this?

I think the original is the proper one, it's named `address` but really what it is is `#<resource_id>@offset into file`, and the result of the match should be that entire regex pattern, not just the address into it. If you see the function `createAddressButton` it better shows what the result should be. Now if this is the best way to do this, probably not.